.github/workflows/deps_lint.yml

```diff
@@ -218,8 +208,40 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -e .[all]
-        pip install -r .github/workflows/requirements-docs.txt --upgrade
+        pip install -r requirements-docs.txt --upgrade
```

Comment:
Why are we adding `--upgrade` to this line while also having a fully specified requirements.txt file? Seems like we're not using the requirements.txt file properly?

Reply:
Another left over from setuptools_scm.